packages/x-license/src/verifyLicense/verifyLicense.ts

```diff
@@ -169,6 +176,15 @@ export function verifyLicense({
     return { status: LICENSE_STATUS.Invalid };
   }
 
+  // 'legacy' is only available for licenses ordered between 2024-06-20 and 2024-07-20
+  if (!license.planVersion || license.planVersion === 'legacy') {
```

Comment:
How about `decodeLicense` always return a `planVersion` (with a default value set for both KV=1 and KV=2 if none is defined in the license)
That way all the code outside of it don't have to do these checks